travis/travis_install_nightly

```diff
@@ -32,6 +32,13 @@ ln -s `which nodejs` $HOME/maintainer-quality-tools/travis/node
 npm install -g less less-plugin-clean-css
 
 
+# Update PhantomJS (v10 compat)
+if [ "${PHANTOMJS_VERSION}" != "OS" ];
```

Comment:
You're gonna kill me for this comment, 😆 but if you are splitting the `if` and `then` in 2 lines, you don't need the `;`